src/constants.js

```diff
@@ -0,0 +1,6 @@
+// eslint-disable-next-line import/prefer-default-export
+// TODO: ATLAS metadata is *not* using the trailing ':'.
+export const VERSION = 'urn:cts:greekLit:tlg0012.tlg001.perseus-grc2:';
+export const INITIAL_PASSAGE = `${VERSION}1.1-1.7`;
+
+export const FETCH_METADATA = 'fetchMetadata';
```

Comment:
May be worth splitting mutation types to a separate file like *mutation-types.js*.

Reply:
Not opposed to this at all, but let's wait and do it in the packaging stuff if we do it, just to get ahead.